starter-analytics-postgres/src/test/groovy/io/micronaut/starter/analytics/postgres/GenerateExcelSpec.groovy

```diff
@@ -0,0 +1,105 @@
+package io.micronaut.starter.analytics.postgres
+
+import io.micronaut.context.annotation.Property
+import io.micronaut.context.annotation.Requires
+import io.micronaut.context.env.Environment
+import io.micronaut.data.model.query.builder.sql.Dialect
+import io.micronaut.http.MediaType
+import io.micronaut.http.annotation.Consumes
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.client.annotation.Client
+import io.micronaut.starter.application.ApplicationType
+import io.micronaut.starter.options.*
+import io.micronaut.test.extensions.spock.annotation.MicronautTest
+import io.micronaut.test.support.TestPropertyProvider
+import jakarta.inject.Inject
+import org.apache.poi.ss.usermodel.DateUtil
+import org.apache.poi.ss.usermodel.Workbook
+import org.apache.poi.xssf.usermodel.XSSFWorkbook
+import org.testcontainers.containers.PostgreSQLContainer
+import spock.lang.AutoCleanup
+import spock.lang.Shared
+import spock.lang.Specification
+
+import java.util.concurrent.CompletableFuture
+
+import static io.micronaut.starter.analytics.postgres.ExcelGenerator.ExcelColumn.*
+
+@Property(name = "spec.name", value = "GenerateExcelSpec")
+@MicronautTest(
+        transactional = false,
+        environments = Environment.GOOGLE_COMPUTE)
```

Comment:
why do we need to be in GCP env?

Reply:
We don't. I had copied that part from an existing test.

I just removed env config from both of them.